Aboutme.js

```diff
@@ -1,8 +1,13 @@
 'use strict';
 
+//yes no questions
+var userScore = 0;
+
+function yesNoQuestion (){
 var answer = prompt('Do you like scary stories?');
```

Comment:
you have all these functions, but you aren't indenting the code blocks inside them.  Everything between the curly braces (lines 7-55) should be indented to another level to indicate they are inside that function, similar to what we do with our if/elses and our loops